src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -278,6 +279,61 @@ protected void printStartupMessage(final ZonedDateTime startDateTime) {
         catch (final Exception e) { /* Unpossible! */ }
     }
 
+    /**
+     * If this tool is either Experimental or Beta, return a warning message advising against use in production
+     * environment.
+     * @param useTerminalColor true if the message should include highlighting terminal colorization
+     * @return a warning message if the tool is Beta or Experimental, otherwise null
+     */
+    protected String getProductionWarning(final boolean useTerminalColor) {
```

Comment:
`getToolStatusWarning()` might be clearer?

Reply:
Done.